rose/server/player_test.py

```diff
@@ -1,15 +1,88 @@
-from . import player
+import pytest
+from rose.common import actions, config
+from rose.server.player import Player
 
 
-def test_in_lane():
-    p = player.Player("A", car=0, lane=0)
-    for x in (0, 1, 2):
-        p.x = x
-        assert p.in_lane()
+# Sample Test Setup
+@pytest.fixture
+def player():
+    return Player("John", "car1", 1)
```

Comment:
I don't think the fixture helps here - we don't have any complex state to manage, and the tests will be more clear if we create the player right in the test as you do in some of the tests.

A fixture can be useful to test with different configurations, like different values of cells_per_player.

Reply:
removed the fixture, IMHO it is a matter of taste because the fixture prevent code repetitions not just for different configurations.